clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -2455,7 +2454,7 @@ private void close(Duration timeout, boolean swallowException) {
             closeTimer.reset(remainingDurationInTimeout);
 
             // This is a blocking call bound by the time remaining in closeTimer
-            LambdaUtils.swallow(() -> fetcher.close(closeTimer), firstException);
+            Utils.swallow(log, " fetcher close", () -> fetcher.close(closeTimer), firstException);
```

Comment:
nit: There is an extra space before `fetcher`.

Reply:
fixed